pkg/minishift/registration/redhat.go

```diff
@@ -90,10 +91,21 @@ func (registrator *RedHatRegistrator) Register(param *RegistrationParameters) er
 				param.Username,
 				minishiftStrings.EscapeSingleQuote(param.Password))
 
+			// prepare docker login command for registry.redhat.io
+			dockerLoginCommand := fmt.Sprintf("docker login --username %s --password %s %s",
+				param.Username,
+				minishiftStrings.EscapeSingleQuote(param.Password),
+				RedhatRegistry)
```

Comment:
`RedhatRegistry` or `RedHatRegistry`... just like `RedHatRegistrator` and the official way of writing the name